state/relation.go

```diff
@@ -249,12 +249,25 @@ func (r *Relation) checkConsumePermission(offerUUID, userId string) (bool, error
 	return true, nil
 }
 
+// DestroyWithForce may force the destruction of the relation.
+func (r *Relation) DestroyWithForce(force bool) error {
```

Comment:
Since this is new, may as well make it return the errors up.

Reply:
Good point. Will adjust this and other, similar places \o/